build.yaml

```diff
@@ -497,6 +497,23 @@ common_cli:
     ${ACCEPTANCE_TEST_MODULE}
     JOB_EXIT_STATUS="$( updateExitStatus $? )"
 
+  run_acceptance_gce_loopback_tests: &run_acceptance_gce_loopback_tests |
+    # We gather the return code but make sure we come out of these tests with 0
+    # we store that code and pass it to the end of the job execution,
+    # as part of the JOB_EXIT_STATUS variable.
+    #
+    # The admin/run-acceptance-tests will provision a flocker cluster with a
+    # single node since the loopback backend doesn't support moving data across
+    # hosts.
+    ${venv}/bin/python admin/run-acceptance-tests \
+    --distribution ${DISTRIBUTION_NAME} --number-of-nodes 1 \
+    --provider gce --dataset-backend loopback --branch ${TRIGGERED_BRANCH} \
```

Comment:
Ooo I hadn't thought of that, nice.

Nothing has ever gone wrong with pretending global variables are arguments to functions before, right?